### PR TITLE
Beta 12 Polishing tasks update

### LINKED ISSUE #989: Beta 12 polishing tasks
**Description**

Beta 12 is missing the following items:

- Update the new roadmap on the landing page
- Add Quickswap, DFYN in that scrolling text for supported dexes
- Add Polygon in the networks 

**AC**

- Roadmap matches the one described in Figma
- Quickswap and DFYN logos appear in the DEX sliders 
- Polygon is added in network list

### PR DESCRIPTION
**Pull request by Mi-Lan**
# Summary

Fixes #989 
- Added updated roadmap, polygon network and routing through new icons

Screenshot 2022-05-17 at 14 47 44
Screenshot 2022-05-17 at 14 47 37
Screenshot 2022-05-17 at 14 47 34

  # To Test

1. Go to landing page and check if everything is according to designs

### CODE REVIEW

File: src/components/LandingPageComponents/layout/Hero.js [hunk 1]
```diff
@@ -75,8 +75,10 @@ const Hero = props => {
     const temporaryArray = []
     for (i = 0, j = HeroContent.heroLogos.length; i < j; i += chunk) {
       temporary = HeroContent.heroLogos.slice(i, i + chunk)
+      console.log(HeroContent.heroLogos.slice(i, i + chunk))
       temporaryArray.push(temporary)
     }
+    console.log(temporaryArray)
```
Comment (adamazad) [inline src/components/LandingPageComponents/layout/Hero.js]: Debugging purposes?

File: src/components/LandingPageComponents/layout/Hero.js [hunk 2]
```diff
@@ -89,6 +91,8 @@ const Hero = props => {
           <div className="hero-content" data-aos="fade-up">
             <h1>{HeroContent.mainText}</h1>
             <ul className="hero-logos-list">
+              <div>hereedeeesssssssee{logosArrays}</div>
+
```
Comment (adamazad) [inline src/components/LandingPageComponents/layout/Hero.js]:
```suggestion
```

File: src/components/LandingPageComponents/layout/Hero.js [hunk 3]
```diff
@@ -99,6 +103,7 @@ const Hero = props => {
                 </div>
               ))}
             </ul>
+
```
Comment (adamazad) [inline src/components/LandingPageComponents/layout/Hero.js:102]:
```suggestion
```

File: src/utils/milestones.js [hunk 4]
```diff
@@ -1,20 +1,16 @@
...
   {
     title: '2021 Q1',
     content: [
-      'DXDAO-supplied liquidity',
+      'Swapr Alpha Launch',
+      'DXdao-supplied liquidity',
       'Swapr on xDai',
       'Eco-Routing Beta',
-      'Uniswap, Sushiswap, Honeyswap, LevinSwap Routing',
+      'Uniswap, Suishiswap Honeyswap, LevinSwap Routing',
```
Comment (adamazad) [inline src/utils/milestones.js]: I think we need to consider the branding of SushiSwap. The official website says Sushi alone. 

https://www.sushi.com/

cc @0xVenky

File: src/utils/milestones.js [hunk 5]
```diff
@@ -28,38 +24,40 @@ export const TimelineData = [
       'Network Switcher v2',
       'Swapr L1-L2 Bridge',
     ],
-    releasteTag: 'BETA',
+    releaseTag: 'BETA',
```
Comment (adamazad) [inline src/utils/milestones.js:27]: Good catch!

File: src/utils/milestones.js [hunk 6]
```diff
@@ -28,38 +24,40 @@ export const TimelineData = [
...
-      'SWPR Governance',
+      'Socket, Omni, Connext  & Xdai Bridge',
+      'Router & Bridge on Polygon',
       'DIY Farming 1.5',
-      'Anyswap/Multichain',
-      'User Dashboard',
-      'LP Governance',
-      'Landing Page',
+      'Advanced Trading',
+      'Curve, 0x & Cow Protocol integration',
```
Comment (adamazad) [inline src/utils/milestones.js:43]:
```suggestion
      'Curve, 0x & CoW Protocol integration',
```

**Review by adamazad — CHANGES_REQUESTED** [review]
Few things and we should be good to go. Great job as always!

**Review by berteotti — APPROVED** [review]

**Review by niemam29 — APPROVED** [review]
Looks good!

**Review by adamazad — APPROVED** [review]

**Pull request merged**